docs/version0.33.md

```diff
@@ -22,9 +22,9 @@
 * Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-# What's new in version 0.33.0
+# What's new in version 0.33.x
```

Comment:
This should say "versions".

Reply:
I'm not sure I agree. It's a bit of a grey area but we already talk about, for example OpenSSL 1.1.x as singular rather than plural (eg in https://eclipse-openj9.github.io/openj9-docs-staging/979/introduction/#cryptographic-operations) and that would look odd if you made it plural. Similarly, we talk about Java version 8 as one release even though it's actually lots of sub-releases (we don't call it version 8.x but we could do).

Reply:
You suggested "OpenJ9 releases 0.33.x support" which considers "0.33.x" to be plural, so it would be inconsistent not to do so here as well.

Reply:
I made that suggestion on this sentence: "OpenJ9 release 0.33.0 and 0.33.1 supports OpenJDK 8, 11, 17, and 18." (I think that sentence got lost in the force push?) Now the sentence reads "OpenJ9 releases 0.33.x support OpenJDK 8, 11, 17, and 18.", it sounds a bit odd and I would change it to "OpenJ9 release 0.33.x supports OpenJDK 8, 11, 17, and 18."